rules/okta_rules/okta_password_accessed.py

```diff
@@ -0,0 +1,39 @@
+from panther_base_helpers import deep_get, lst_iter
+
+
+def rule(event):
+    global target_user
+    global target_app_names
+    # event['target'] = [{...}, {...}, {...}]
+    target_user = lst_iter(
+        event.get("target", [{}]), 'alternateId', 'Type', 'AppUser')
+    target_app_names = target_app_names = lst_iter(
+        event.get("target", [{}]), 'alternateId', 'Type', 'AppInstance')
+
+    if (
+        event.get("eventType") != "application.user_membership.show_password"
+        or not event.udm("actor_user")
```

Comment:
It's not clear to me why the second part of this conditional is here. I don't see where we're using `event.udm("actor_user")` later in this detection.